packages/main/src/ListItem.js

```diff
@@ -94,6 +94,8 @@ class ListItem extends ListItemBase {
 	onBeforeRendering() {}
 
 	onkeydown(event) {
+		super.onkeydown(event);
```

Comment:
Why is this needed now and wasn't before?

Reply:
the logic was handled in psuedo events `onsap...` which was propagated from the `ListItem` to `ListItemBase`. As we are not having these event anymore, we should always call parent's `keydown` if we want to have propagation in the inheritance chain